backend/src/common/database/enums/database-column-name.enum.ts

```diff
@@ -40,6 +40,11 @@ enum DatabaseColumnName {
     TEMPLATE_ID = 'template_id',
     TEMPLATE_SETTINGS = 'template_settings',
     OWNER_ID = 'owner_id',
+    NEW_ID = 'new_id',
+    NEW_USER_ID = 'new_user_id',
+    NEW_PROFILE_ID = 'new_profile_id',
+    NEW_RESUME_ID = 'new_resume_id',
+    NEW_TEMPLATE_ID = 'new_template_id',
 }
 
```

Comment:
it seems like it is running with the current version, however, I'm unable to rollback, have you tried to do so?

Reply:
NEW_USER_ID  and so on are are temporary names that are immediately renamed to id. 
This is necessary so that we can save existing  id values in the database.
We can't just add an automatic value generation for id because of restrictions. 
Therefore, the algorithm is as follows:
- create a new column new_id;
- drop the restriction premary key and foreigh key;
- copy values from the previous id to the new one;
- then new_id is renamed to id;

Reply:
waiting for a response regarding the rollback